config/api/src/main/java/org/jboss/windup/config/RuleSubset.java

```diff
@@ -190,7 +190,7 @@ public void perform(Rewrite rewrite, EvaluationContext context)
             throw new IllegalArgumentException("Rewrite must be an instanceof GraphRewrite");
 
         /*
-         * Highly optimized loop - for performance reasons. Think before you change this!
+         * Highly optimized loop - for performance reasons. Think before you change this! (lincolnthree)
          */
```

Comment:
It's a highly optimized loop that we only go once through for each rule :)
This is a copy & paste from Rewrite IIRC, and should be refactored somehow so that we don't need to have a clone but rather an override.